packages/server/src/api/customers/customers.service.ts

```diff
@@ -222,81 +223,24 @@ export class CustomersService {
       ).exec();
     }
 
-    await this.dataSource.transaction(async (transactionManager) => {

```

Comment:
I do not think this logic is the same in journeysService.enrollCustomer I prefer to double check because I can not find logic for static status